cprover_bindings/src/goto_program/typ.rs

```diff
@@ -343,6 +347,7 @@ impl Type {
             IncompleteStruct { .. } => unreachable!("IncompleteStruct doesn't have a sizeof"),
             IncompleteUnion { .. } => unreachable!("IncompleteUnion doesn't have a sizeof"),
             InfiniteArray { .. } => unreachable!("InfiniteArray doesn't have a sizeof"),
+            Integer => st.machine_model().int_width,
```

Comment:
Are we sure that's true? Is there documentation for this?

Reply:
Looks like it's not (see @tautschnig comments).

Reply:
Done.